### ISSUE TITLE
parmest: allow user to specify path to ipopt executable

### ISSUE THREAD

**Issue by adowling2** [Pyomo/pyomo#1566]
This would be a nice enhancement for folks with multiple versions of Ipopt installed. Alternately, we could let the user provide an Ipopt solver object.

**Comment by DLWoodruff** [comment]
I'll try to add this feature.

**Comment by adowling2** [comment]
@djlaky Is this issue now obsolete with the parmest refactor?

**Comment by mrmundt** [comment]
Is this good to close?

**Comment by djlaky** [comment]
@mrmundt Yes, I’ve noted this issue about specifying solvers to be added in a later PR (and is included in our DoE issue). In other words, the issue raised here is no longer a problem.
